form.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+    
+        <section class="container">
+            
+            <fieldset>
+                <legend>Personal information</legend>
+
+                <div class="form-field">
+
+                    <label>
+                        Surname:
+                        <input type="text" name="surmame" required> 
```

Comment:
```suggestion
                        <input
                          type="text"
                          name="surmame"
                          required
                        > 
```
Коли атрибутів 3 і більше - прийнято кожен писати з нового рядка